elasticsearch/Dockerfile.centos7

```diff
@@ -52,7 +52,7 @@ ADD probe/ ${ES_HOME}/probe/
 ADD init.sh run.sh prep-install.${RELEASE_STREAM} install.sh ${HOME}/
 COPY utils/** /usr/local/bin/
 
-ARG OSE_ES_URL
+ARG OSE_ES_URL=https://github.com/jcantrill/openshift-elasticsearch-plugin/releases/download/openshift-elasticsearch-plugin-5.6.16.2/openshift-elasticsearch-plugin-5.6.16.2.zip
```

Comment:
should this still be pointing at your repo?

Reply:
yes.  testing that it passes before actually releasing and merging